crates/cxx-qt-gen/src/syntax/foreignmod.rs

```diff
@@ -37,7 +37,7 @@ pub(crate) fn foreign_mod_to_foreign_item_types(
         .filter_map(|item| match foreign_item_to_type(item) {
             Ok(Some(value)) => Some(Ok(value)),
             Ok(None) => None,
-            Err(err) => Some(Err(err)),
+            Err(err) => Some(Err(err)), // Hard to cover since foreign_item_to_type never returns error
```

Comment:
to reach the `Unsupported verbatim input in ForeignItem` error we just need to construct a Verbatim input that doesn't start with `type`.

A fn one is quite simple to do https://github.com/dtolnay/syn/blob/94d74d52ca8ec4fb46317f261edd678f9231e178/src/item.rs#L1825

So basically this should then hit that error case, as it has a Brace
```rust
extern "RustQt" {
   fn my_function() {}
}
```